src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -337,11 +337,13 @@ public final class JavadocTokenTypes {
      * <pre>{@code @author Baratali Izmailov}</pre>
      * <b>Tree:</b>
      * <pre>{@code
-     *   |--JAVADOC_TAG[3x0] : [@author Baratali Izmailov]
-     *       |--AUTHOR_LITERAL[3x0] : [@author]
-     *       |--WS[3x7] : [ ]
-     *       |--DESCRIPTION[3x8] : [Baratali Izmailov]
-     *           |--TEXT[3x8] : [Baratali Izmailov]
+     *   --JAVADOC_TAG -> JAVADOC_TAG
+     *      |--AUTHOR_LITERAL -> @author
+     *      |--WS ->
+     *      `--DESCRIPTION -> DESCRIPTION
+     *          |--TEXT -> Baratali Izmailov
+     *          |--NEWLINE -> \r\n
```

Comment:
Please remove this line, it windows os specific.

Reply:
Done

Reply:
I meant to remove `|--NEWLINE -> \r\n`, but it is ok, we will be perfectly exact when we will automate this content generations